src/Advection/vector_invariant_advection.jl

```diff
@@ -116,27 +163,65 @@ Base.show(io::IO, a::VectorInvariant{N, FT}) where {N, FT} =
               "$(a.vertical_scheme isa AbstractUpwindBiasedAdvectionScheme ? 
               "\n └── upwinding treatment: $(a.upwinding)" : "")")
 
+#####
+##### Convenience for WENO Vector Invariant
+#####
+
+#                           VectorInvariant{N,     FT,    M,     Z,      ZS,    V,      K (kinetic energy gradient scheme)
+const WENOVectorInvariant = VectorInvariant{<:Any, <:Any, <:Any, <:WENO, <:Any, <:WENO, <:WENO}
+
+"""
+    function WENOVectorInvariant(; upwinding = nothing,
+                                   multi_dimensional_stencil = false,
+                                   weno_kw...)
+
+"""
+function WENOVectorInvariant(; upwinding = nothing,
+                               vorticity_stencil = VelocityStencil(),
+                               multi_dimensional_stencil = false,
+                               weno_kw...)
+
+    weno = WENO(; weno_kw...)
```

Comment:
most of the time we would like a higher order for vorticity. Can we have a `vorticity_order` keyword argument?

Reply:
what should be the default order?

Reply:
I found that 9 for vorticity and 5 for the other gave good results with high enough resolution.

Reply:
But not order 9 for the others?

Reply:
with 9 for the others, the scheme becomes heavy without really affecting the solution much.
It is also nice to keep a lower order scheme for the vertical advection because it is more of a "difficult" term when using low vertical resolution typical of a hydrostatic model.